### PR TITLE
Feature/2070 modify dx table

### PR DESCRIPTION
**Pull request by evans-g-crsj**
Side note not related to the feature: while working on this page I noticed that sometimes it displays one table and a few seconds later one or two other table appears (qa was slow at that time). There is no loader indicating that a data fetching is occurring.

### CODE REVIEW

File: src/components/EntityPage/Participant/ParticipantClinical.js [hunk 1]
```diff
@@ -43,8 +46,8 @@ class ParticipantClinical extends React.Component {
       });
     }
 
-    let diagnoses = get(this.props.participant, 'diagnoses.hits.edges', []).map(ele =>
-      Object.assign({}, get(ele, 'node', {})) //copy obj
+    let diagnoses = get(this.props.participant, 'diagnoses.hits.edges', []).map(
+      ele => Object.assign({}, get(ele, 'node', {})), //copy obj
```
Comment (jecos) [inline src/components/EntityPage/Participant/ParticipantClinical.js]: Remove comma

Comment (jberube) [reply src/components/EntityPage/Participant/ParticipantClinical.js]: Auto-format by Prettify.

It makes the code uniform, but it sometimes makes a :hankey:

Comment (evans-g-crsj) [reply src/components/EntityPage/Participant/ParticipantClinical.js]: good point

File: src/components/EntityPage/Participant/ParticipantClinical.js [hunk 2]
```diff
@@ -254,5 +323,8 @@ class ParticipantClinical extends React.Component {
     );
   }
 }
-//: {get(participant, "family.family_composition.hits.edges[0].node.composition", "trio")}
-export default withRouter(ParticipantClinical);
+
+const ParticipantClinicalWithRouter = withRouter(ParticipantClinical);
+const ParticipantClinicalWithRouterAndTheme = withTheme(ParticipantClinicalWithRouter);
+
+export default ParticipantClinicalWithRouterAndTheme;
```
Comment (jberube) [inline src/components/EntityPage/Participant/ParticipantClinical.js]: You can simply do (if you prefer):
```js
export default withTheme(withRouter(ParticipantClinical));
```

**Pull request merged**